partner_invoicing_mode/models/sale_order.py

```diff
@@ -70,12 +70,16 @@ def _generate_invoices_by_partner(self, saleorder_ids):
         )
         if not sales:
             return "No sale order found to invoice ?"
-        invoices = sales._create_invoices(
-            grouped=sales[0].partner_invoice_id.one_invoice_per_order, final=True
```

Comment:
@rafaelbn FYI This line is not correct and fixed by using the partition function. The current code takes the first sale order property (and don't distinguish each one).